package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.16",
   "description": "An exceptionally fast, thorough and tiny unused-CSS cleaner",
   "main": "./dist/dropcss.cjs.js",
+  "types": "./src/types.d.ts",
```

Comment:
this also needs to be added to the `files` array below

Reply:
Great catch, ty!